src/main/java/de/tum/in/www1/artemis/domain/metis/conversation/GroupChat.java

```diff
@@ -40,4 +45,22 @@ public String getName() {
     public void setName(String name) {
         this.name = name;
     }
+
+    @Override
+    public String getHumanReadableNameForReceiver(User sender) {
+        if (StringUtils.isBlank(getName())) {
+            final String generatedName = getConversationParticipants().stream().map((participant) -> participant.getUser().getName()).collect(Collectors.joining(", "));
+
+            // The name should be human-readable, so we limit it for very long lists and add "…" to hint that the string is not complete.
+            if (generatedName.length() >= Constants.GROUP_CONVERSATION_HUMAN_READABLE_NAME_LIMIT) {
+                return generatedName.substring(0, Constants.GROUP_CONVERSATION_HUMAN_READABLE_NAME_LIMIT) + "…";
```

Comment:
Idea: Maybe not split this after a fixed amount of chars, but after the last name before reaching the limit.

So Instead of "Lucas Welscher, Tim Ortel, Jan-Phi..." it should be just "Lucas Welscher, Tim Ortel, ..."